src/main/resources/db/changelog/db.changelog-RDM-9052.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    
+    <changeSet id="RDM-9052-1" author="henry">
+	    <preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">SELECT count(c.relname) FROM pg_class c WHERE c.relkind = 'S' and c.relname = 'user_profile_audit_id_seq'</sqlCheck>
+	    </preConditions>
```

Comment:
can we verify whether this precondition is really needed? 
Maybe we are able to get rid of it and Liquibase is smart enough to handle things correctly and set the column to autoincrement even if the sequence already exists?
Worth giving it a try since if that works we'd be able to simplify this code quite a lot

Reply:
At least we need to check (precondition) if table exists or not.. otherwise <addAutoIncrment... is not going to work..

Reply:
Pre condition is really required. As per the definition of serial type, the sequence is created if the table has serial type declared. The existence of sequence without the table only happens when the db is corrupt or the sequence is created by hand. considering the impending flyway changes. doing other checks little bit overboard.